### PR TITLE
Handle coordinates calculation for non-default screen orienation, which has been fixed since iphoneos SDK 11.0

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
XCTest has a bug when tap coordinates are inverted if device orientation differs from portrait, which has been fixed since SDK version 11.0 (Xcode9+). This PR checks the actual SDK version used to compile WDA to determine whether to apply coordinates calculation workaround.

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
Linking https://github.com/facebook/WebDriverAgent/issues/705

**Comment by cosminstirbu** [comment]
Hello,

We'd really appreciate if this can get reviewed and merged as it is causing a high number of our tests to fail.

Thank you,
Cosmin

**Comment by mykola-mokhnach** [comment]
I checked the general functionality with integration tests we already have for _fb_tap_ method. Would you like to cover other methods with unit tests?

File: WebDriverAgentTests/UnitTests/FBSDKVersionTests.m [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+@property (nonatomic, readonly) NSString *higherSDKVersion;
+@end
+
+@implementation FBSDKVersionTests
+
+- (void)setUp
+{
+  [super setUp];
+  NSDictionary *bundleDict = [[NSBundle mainBundle] infoDictionary];
+  [bundleDict setValue:@"11.0" forKey:@"DTSDKName"];
```
Comment (marekcirkos) [inline WebDriverAgentTests/UnitTests/FBSDKVersionTests.m]: Hacky like hell :).
Easier approach might be having two layer API, one where you pass current version explicitly (that one you test) and second convenience one that passes current SDK version.

Tho we can merge this one and fix it if there is a problem.

**Pull request closed without merging**